features/machine/autoscaler.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Cluster Autoscaler Tests
+
+  # @author jhou@redhat.com
+  # @case_id OCP-28108
+  @admin
+  @destructive
+  Scenario: Cluster should automatically scale up and scale down with clusterautoscaler deployed
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    Given I store the number of machines in the :num_to_restore clipboard
+    And admin ensures the node number is restored to "<%= cb.num_to_restore %>" after scenario
+
+    Given I clone a machineset named "machineset-clone"
+    And admin ensures "machineset-clone" machineset is deleted after scenario
+
+    # Create clusterautoscaler
+    Given I use the "openshift-machine-api" project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/cloud/cluster-autoscaler.yml |
+    Then the step should succeed
+    And 1 pods become ready with labels:
+      | cluster-autoscaler=default,k8s-app=cluster-autoscaler |
+    And I register clean-up steps:
```

Comment:
Perhaps register the clean-up earlier. E.g, if the above pod failed to become ready, then the clean-up is not registered successfully.

Reply:
Thanks, addressed that.